modules/web/src/com/haulmont/cuba/web/exception/ExceptionDialog.java

```diff
@@ -349,7 +362,7 @@ public void setStackTraceVisible(boolean visible) {
         }
     }
 
-    public void sendSupportEmail(String message, String stackTrace) {
+    public void sendSupportEmail(String message, String stackTrace, String userMessage) {
```

Comment:
It's a breaking change. Please add a new method with 3 arguments.

Reply:
I added cammit - saved the method with two arguments and added a new method with three arguments.

Reply:
Could you do it in a more DRY manner? 
```
public void sendSupportEmail(String message, String stackTrace) {
    sendSupportEmail(message, stackTrace, "");
}
```